docs/nitpick-exceptions

```diff
@@ -72,58 +50,33 @@ py:class astropy.wcs.wcsapi.fitswcs.custom_ctype_to_ucd_mapping
 # numpy inherited docstrings
 py:obj dtype
 py:obj a
-py:obj a.size == 1
 py:obj n
 py:obj ndarray
 py:obj args
 
 # other classes and functions that cannot be linked to
-# NUMPY_LT_2_0: numpy 2.0 changed numpy.core to numpy._core
```

Comment:
As for `numpy.core` vs `numpy._core` here, won't this come back to haunt us when numpy 2.0 is released and RTD picks it up? 🤔

Reply:
I'd rather not worry about this unless there is proof this is worth worrying about.

Reply:
We might just end up reverting a few of these lines when numpy 2.0 is released, so what is the point of removing it too soon? Should this PR wait till after numpy 2.0 is released?